pom.xml

```diff
@@ -298,7 +298,7 @@
       <dependency>
         <groupId>${project.groupId}</groupId>
         <artifactId>netty-tcnative</artifactId>
-        <version>1.1.33.Fork10</version>
+        <version>1.1.33.Fork11</version>
```

Comment:
@normanmaurer - Do you think we could cut a netty-tcnative release for this soon?